__tests__/app.test.js

```diff
@@ -70,3 +70,55 @@ describe("GET /api/reviews/:review_id", () => {
     });
   });
 });
+
+describe("PATCH /api/reviews/:review_id", () => {
+  test("status: 202, in/decrements votes column for the given review_id by the value of inc_votes, responds with updated review", async () => {
+    const { body } = await request(app)
+      .patch("/api/reviews/2")
+      .send({ inc_votes: -1 })
+      .expect(202);
```

Comment:
`202 Accepted` doesn't mean that the request has been done - just that the server has accepted it!

you might want to go for the tried and true `200` here

[read more](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202)

Reply:
Changed it to 200, thanks. So we'd use 202 if the server had queued the patch to do later, or something along those lines?